lib/graphql_generator.dart

```diff
@@ -9,25 +9,21 @@ import 'package:build/build.dart';
 import 'package:graphql_generator/annotation.dart';
 import 'package:graphql_generator/generator/code_generator.dart';
 import 'package:graphql_generator/generator/model.dart';
+import 'package:graphql_generator/schema_fetcher.dart';
 import 'package:http/http.dart' as http;
 import 'package:source_gen/source_gen.dart';
 
 class GraphQLGenerators extends GeneratorForAnnotation<GQLGenerator> {
-  var url;
-  var headerToken;
-  String namespace = "";
-
-  String mutationType = "";
-
-  Map<String, DartType> types = {};
-  Map<String, String> fragments = {};
-
   ObjectType mutation = new ObjectType();
 
   @override
   Future<String> generateForAnnotatedElement(
       Element element, ConstantReader annotation, BuildStep buildStep) async {
-    url = annotation.read('url').stringValue;
+    var url = annotation.read('url').stringValue;
```

Comment:
Good idea. Moving the variables in here means more encapsulated code and less shared state.